test_data/config.yaml

```diff
@@ -0,0 +1,9 @@
+credentials:
```

Comment:
What are your thoughts on badly formatted YAML files? If I'm reading the code right the line-numbers won't be correct but maybe we decide it doesn't matter that much.

e.g.
```yaml
repos: [
    {repo: ..., sha: ..., hooks: [{id: ...}]},
]
```